src/framework/services/filter/Filter.spec.js

```diff
@@ -58,6 +58,26 @@ describe('Filter', () => {
         const filter = new Filter(filterOption, normalizedComparisonValues);
         expect(filter.humanizeComparisonValue()).toBe('5, 6');
       });
+
+      it('returns joined string with DATE_RANGE comparisonType', () => {
+        const filterOption = {
+          comparisonType: ComparisonTypes.DATE_RANGE,
+        };
+        const normalizedComparisonValues = {
+          startDate: new Date(1989, 10, 30),
+          endDate: new Date(2019, 10, 30),
+        };
+        const filter = new Filter(filterOption, normalizedComparisonValues);
+        // The PhantomJS toLocaleDateString does not have the same default options as chrome
+        expect(filter.humanizeComparisonValue()).toBe(
+          `${
+            normalizedComparisonValues.startDate.toLocaleDateString('de-DE')
+          } - ${
+            normalizedComparisonValues.endDate.toLocaleDateString('de-DE')
+          }`
+        );
+        // .toBe('30.11.1989 - 30.11.2019');
```

Comment:
after talking with alex, im ok with having this comment more clear next to the one talking about phantomJs or removing it